products/sonarqube.md

```diff
@@ -18,25 +18,25 @@ auto:
 
 releases:
 -   releaseCycle: "9"
-    releaseDate: 2021-07-05
+    releaseDate: 2021-07-02
     support: true
     eol: false
     latest: "9.8.0"
     latestReleaseDate: 2022-12-19
 
 -   releaseCycle: "8"
-    releaseDate: 2019-10-16
+    releaseDate: 2020-05-08
```

Comment:
Note correct: https://community.sonarsource.com/t/sonarqube-8-0-released/15540

Reply:
Might be worth iterating over https://downloads-cdn-eu-central-1-prod.s3.eu-central-1.amazonaws.com/?delimiter=/&prefix=Distribution%2Fsonarqube%2F (+https://downloads-cdn-eu-central-1-prod.s3.eu-central-1.amazonaws.com/?delimiter=/&prefix=Distribution%2Fsonarqube%2F&marker=Distribution%2Fsonarqube%2Fsonarqube-9.3.0.51899.zip.sha1 for pagination) instead.

Docker images were introduced quite late, both in this case and others, for it to be effective at tracking releases.

Reply:
Unfortunately it will not be possible: `LastModifiedDate` is not the release date (it is always 2022-02-16).

It would be best to ignore `releaseDate` if we don't have the 8.0.0 version IMHO (which is why I create #2154).